dbms/src/DataStreams/WindowBlockInputStream.cpp

```diff
@@ -678,7 +799,7 @@ RowNumber WindowTransformAction::getPreviousRowNumber(const RowNumber & row_num)
     }
 
     --prev_row_num.block;
-    assert(prev_row_num.block - first_block_number < window_blocks.size());
+    assert(static_cast<Int64>(prev_row_num.block - first_block_number) < static_cast<Int64>(window_blocks.size()));
```

Comment:
Does `static_cast<Int64>(UInt64 - UInt64)` make sense?
Or should it be changed to `static_cast<Int64>(UInt64) - static_cast<Int64>(UInt64)`

Reply:
> Does `static_cast<Int64>(UInt64 - UInt64)` make sense? Or should it be changed to `static_cast<Int64>(UInt64) - static_cast<Int64>(UInt64)`

okk